src/Interfaces/AbstractZonotope.jl

```diff
@@ -884,6 +885,31 @@ function _remove_redundant_generators_1d(G::AbstractMatrix)
     return hcat(g)
 end
 
+"""
+    remove_small_generators(Z::AbstractZonotope, tol::Real=1e-4; p::Real=Inf)
+    
+Remove generators with norm smaller than a given tolerance.
+
+# Input
+
+- `Z` -- an `AbstractZonotope`
+- `tol` -- (default: `1e-4`) threshold below which a generator column is considered negligible
+- `p`  -- (optional, default: `Inf`) norm
+
+# Output
+
+A zonotope containing only the generators whose ``p``-norm is at least `tol`
+"""
+function remove_small_generators(Z::AbstractZonotope, tol::Real=1e-4, p::Real=Inf)
```

Comment:
Maybe it is better to just implement this method for `Zonotope`. I know that `remove_redundant_generators` is implemented for `AbstractZonotope`, but I am not so happy with that anymore.